<?xml version="1.0"?>
<launch>

	<!-- start platform node -->
	<node pkg="cob_platform" type="cob_platform" name="cob_platform" cwd="node" output="screen"/>

	<!-- Axes definition -->
	<param name="axis_linear_x" value="1" type="int"/>
	<param name="axis_linear_y" value="0" type="int"/>
	<param name="axis_angular_z" value="2" type="int"/>
	<param name="scale_linear" value="0.2" type="double"/>
	<param name="scale_angular" value="0.1" type="double"/>

	<!-- cob teleop platform node -->
	<node pkg="cob_teleop" type="cob_teleop_platform_joy" name="teleop_platform"/>

	<!-- joy node -->
	<include file="$(find cob_joy)/ros/launch/$(env ROBOT).launch" />

</launch>
